src/page-response-impact/RtSummaryStats.tsx

```diff
@@ -72,14 +102,15 @@ const RtSummaryStats: React.FC<Props> = ({ rtData }) => {
   const averageRtReductionAcrossFacilities =
     rtStats.averageRtReductionAcrossFacilities(facilitiesRtRecords) || 0;
 
-  const numFacilitiesWithRtGt1 = totalFacilities - numFacilitiesWithRtLt1;
+  const numFacilitiesWithRtGt1 =
+    totalFacilitiesInScenario - numFacilitiesWithRtLt1;
```

Comment:
i am still going to object to this because this is not the number of facilities with Rt greater than 1 ... would call it like `numFacilitiesRemainder` or something else not too wordy that doesn't make any false assertions about its contents

Reply:
Oh yeah, that makes sense--I misunderstood which variable to rename. I'll update this now.